intercept.go

```diff
@@ -115,14 +115,24 @@ func analyzeVideo(videoID string) (reason string, suspicious bool) {
 			// TODO: make viewcount threshold configurable
 			appendStringComma(&reason, "Video has less than 10000 views")
 			suspicious = true
-
 		}
 
 		if video.Statistics.CommentCount < 50 {
 			appendStringComma(&reason, "Video has less than 50 comments")
 			suspicious = true
 		}
-
+		
+		if video.Status.UploadStatus != "uploaded" {
+			appendStringComma(&reasons, "Video is not uploaded")
```

Comment:
Hier fände ich cool wenn der tatsächliche UploadStatus in der Fehlermeldung wäre, z.B.
"Video status is \"" + video.Status.UploadStatus + "\""

https://developers.google.com/youtube/v3/docs/videos#status.uploadStatus